lldb/include/lldb/ValueObject/DILEval.h

```diff
@@ -61,7 +61,15 @@ class Interpreter : Visitor {
   void SetContextVars(
       std::unordered_map<std::string, lldb::ValueObjectSP> context_vars);
 
+  bool AllowSideEffects() const { return m_allow_side_effects; }
+
+  void SetAllowSideEffects(bool allow_side_effects) {
+    m_allow_side_effects = allow_side_effects;
+  }
+
```

Comment:
We never actually un-set allowing side effects anywhere that I can see, so this code seems maybe  unnecessary? (I.e. "AllowSideEffects()" always returns 'true' for us).